ksql-serde/src/main/java/io/confluent/ksql/serde/delimited/KsqlDelimitedTopicSerDe.java

```diff
@@ -16,13 +16,40 @@
 
 package io.confluent.ksql.serde.delimited;
 
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.common.serialization.Serializer;
+import org.apache.kafka.connect.data.Schema;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
+import io.confluent.ksql.GenericRow;
 import io.confluent.ksql.serde.DataSource;
 import io.confluent.ksql.serde.KsqlTopicSerDe;
+import io.confluent.ksql.util.KsqlConfig;
 
 
 public class KsqlDelimitedTopicSerDe extends KsqlTopicSerDe {
 
   public KsqlDelimitedTopicSerDe() {
     super(DataSource.DataSourceSerDe.DELIMITED);
   }
+
+  @Override
+  public Serde<GenericRow> getGenericRowSerde(Schema schema, KsqlConfig ksqlConfig,
```

Comment:
tests for these methods?

Reply:
Yes, will have to add tests in separate PR.